### DISCUSSION TITLE
Velero installs but cannot see Azure storage account

### DISCUSSION THREAD

**Discussion by RichardDuaneWolford** [vmware-tanzu/velero#3869] [Community support Q&A]
I'm deploying Velero using the CLI with this command:

`helm install velero vmware-tanzu/velero --namespace velero --set-file credentials.secretContents.cloud=./credentials-velero --set configuration.provider=azure --set configuration.backupStorageLocation.name=azure --set configuration.backupStorageLocation.bucket='velero' --set configuration.backupStorageLocation.config.resourceGroup=aks-testing-prod-003 --set configuration.backupStorageLocation.config.storageAccount=aks-testing-prod-003 --set snapshotsEnabled=true --set deployRestic=true --set configuration.volumeSnapshotLocation.name=azure --set image.repository=velero/velero --set image.pullPolicy=Always --set initContainers[0].name=velero-plugin-for-microsoft-azure --set initContainers[0].image=velero/velero-plugin-for-microsoft-azure:master --set initContainers[0].volumeMounts[0].mountPath=/target --set initContainers[0].volumeMounts[0].name=plugins`

The pods come up just fine, velero and restic. But when I try to create a backup, it submits successfully but then fails with this error. Any help would be appreciated.

```
Name:         mybackup
Namespace:    velero
Labels:       velero.io/storage-location=azure
Annotations:  velero.io/source-cluster-k8s-gitversion=v1.19.11
              velero.io/source-cluster-k8s-major-version=1
              velero.io/source-cluster-k8s-minor-version=19

Phase:  Failed (run `velero backup logs mybackup` for more information)

Errors:    0
Warnings:  0

Namespaces:
  Included:  *
  Excluded:  <none>

Resources:
  Included:        *
  Excluded:        <none>
  Cluster-scoped:  auto

Label selector:  <none>

Storage Location:  azure

Velero-Native Snapshot PVs:  auto

TTL:  720h0m0s

Hooks:  <none>

Backup Format Version:  1.1.0

Started:    2021-06-10 16:32:58 +0000 UTC
Completed:  <n/a>

Expiration:  2021-07-10 16:32:58 +0000 UTC

Velero-Native Snapshots: <none included>
```

**Comment by RichardDuaneWolford** [comment]
Does anyone have any idea?  The backup location/verified are both "Phase" or something like that, it never comes up.

**Comment by KrishnamoorthyRamalingam** [comment]
Facing the same issue ... kind of struck here, without much support 
time="2021-08-09T12:45:45Z" level=error msg="Error getting a backup store" backup-storage-location=default controller=backup-storage-location error="rpc error: code = Unknown desc = unable to get all required config values: the following keys do not have values: storageAccount" error.file="/go/src/velero-plugin-for-microsoft-azure/velero-plugin-for-microsoft-azure/object_store.go:260" error.function="main.(*ObjectStore).Init" logSource="pkg/controller/backup_storage_location_controller.go:100"

**Reply by KrishnamoorthyRamalingam** [reply]
It is resolved now .. After turning off " snapshotsEnabled: false " in values.yml 

Thank you for checking.

**Comment by benjaminxie** [comment]
Could you paste the output for `velero backup logs mybackup`?
